packages/api-client/src/components/CommandPalette/CommandPaletteImport.vue

```diff
@@ -31,8 +39,16 @@ const { open: openSpecFileDialog } = useFileDialog({
 
 const handleSubmit = async () => {
   if (specUrl.value) {
-    await importSpecFromUrl(specUrl.value)
-    emits('close')
+    try {
+      await importSpecFromUrl(specUrl.value)
+      toast('Import successful', 'info')
+      emits('close')
+    } catch (error) {
+      console.error('the error ', error)
+      const errorMessage =
+        error instanceof Error ? error.message : 'Unknown error'
```

Comment:
`error?.message || 'Unknown Error'` is probably a bit easier here.

Reply:
sure is but it was giving me a ts error, i'll update